py/client/pydeephaven/session.py

```diff
@@ -153,12 +227,27 @@ def _fetch_fields(self):
 
     def _connect(self):
         with self._r_lock:
-            self.grpc_channel, self.session_token, self._timeout = self.session_service.connect()
-            self.is_connected = True
+            try:
+                self._flight_client = paflight.connect(location=(self.host, self.port), middleware=[
+                    _DhClientAuthMiddlewareFactory(self)])
+                self._auth_handler = _DhClientAuthHandler(self)
+                self._flight_client.authenticate(self._auth_handler)
```

Comment:
Is the initial auth only used once?  I think it is only being used once.  @niloc132 mentioned problems if it gets reused.

Reply:
yes.